monkey/monkey_island/cc/next_ui/src/app/(auth)/registration/page.tsx

```diff
@@ -67,66 +71,75 @@ const RegisterPage = () => {
-                        sx={{
-                            marginTop: 8,
-                            display: 'flex',
-                            flexDirection: 'column',
-                            alignItems: 'center'
-                        }}>
-                        <Avatar sx={{ m: 1, bgcolor: 'primary.main' }}>
-                            <LockOutlinedIcon />
-                        </Avatar>
-                        <Typography component="h1" variant="h5">
-                            Register
-                        </Typography>
-                        <Box
-                            component="form"
-                            onSubmit={handleSubmit}
-                            sx={{ mt: 1 }}>
-                            <TextField
-                                margin="normal"
-                                required
-                                fullWidth
-                                id="username"
-                                label="username"
-                                name="username"
-                                autoComplete="username"
-                                value={registerFormValues.username}
-                                onChange={handleRegisterFormValueChange}
-                                autoFocus
-                            />
-                            <TextField
-                                margin="normal"
-                                required
-                                fullWidth
-                                name="password"
-                                label="Password"
-                                type="password"
-                                id="password"
-                                autoComplete="current-password"
-                                value={registerFormValues.password}
-                                onChange={handleRegisterFormValueChange}
-                            />
-                            <Button
-                                type="submit"
-                                fullWidth
-                                variant="contained"
-                                sx={{ mt: 3, mb: 2 }}>
-                                Register
-                            </Button>
-                            {/* @ts-ignore */}
-                            {isError && Array.isArray(error) && (
-                                <ErrorList errors={error} />
-                            )}
-                        </Box>
-                    </Box>
+                <Container component="main" maxWidth="xs" sx={containerStyle}>
+                    <Stack direction="column" alignItems={'center'} spacing={2}>
+                        {renderFormCard()}
+                    </Stack>
                 </Container>
             </>
         );
     };
+    const renderFormCard = () => {
+        return (
+            <Card variant="outlined" sx={cardStyle(theme)}>
+                <BrandHeader sx={{ height: '50px' }} />
```

Comment:
Can we put this in `style.ts`?

Reply:
When it comes to 1-2 style values I'd rather have them inline. This way you don't need to jump files trying to correlate which style is applied where for one margin.